engine/gui/src/gui.cpp

```diff
@@ -3646,22 +3659,19 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
         Animation animation;
         uint32_t animation_index = 0xffffffff;
 
-        // Remove old animation for the same property
-        for (uint32_t i = 0; i < scene->m_Animations.Size(); ++i)
+        uint32_t* animation_index_ptr = scene->m_ValueToAnimationIndex.Get(PtrToUint(value));
```

Comment:
Using a hashtable was a huge time saver.
Going from ~150ms, to ~95ms in a very large scene.